docs/building_on_ubuntu_20_04.md

```diff
@@ -0,0 +1,62 @@
+# Building on Ubuntu 20.04
+
+This documents step by step on how to compile and put into use the software `mod_tile` and `renderd`.
+Please see our [Continous Integration script](../.github/workflows/build-and-test-ubuntu-20-04.yml) for more detail.
+
+```shell
+#!/usr/bin/env bash
+
+# Update installed packages
+sudo apt update && apt upgrade --yes
```

Comment:
Should this also be applied in `.github/actions/apt/update/action.yml`? I forgot about the `command` differences between `yum` and `apt` for updating packages.

Reply:
Yes, probably. I added it there.

Reply:
It adds up 3 minutes to each job (including the linter). I think it is good to have it, even though we are slower. Wdyt?

Reply:
Yeah, it's not really required, especially if it takes that long. You could even try removing the `yum update` step from the `provision` action to speed things up as well.